state-chain/pallets/cf-broadcast/src/tests.rs

```diff
@@ -1191,25 +1287,30 @@ fn broadcast_retries_will_not_be_overwritten_during_safe_mode() {
 			assert_ok!(Broadcaster::transaction_failed(RuntimeOrigin::signed(0u64), broadcast_id));
 
 			// On safe mode next block, storage will be re-added to this target block.
-			let next_block = System::block_number() + 1u64;
-			target = next_block +
+			let statechain_next_block = System::block_number() + 1u64;
```

Comment:
generally if there's a context where state chain and external chain blocks are used then we use "block" and "chain block" for state chain and external chain blocks respectively, so we should be consistent here

Reply:
Ok, changing. (The convention is quite confusing though, isn't it...)

Reply:
only when you're not used to it ;) - but yeah i agree, the options are a little limited without having huge type names etc.

Reply:
I guess then the new constant I added should be called `SafeModeChainBlockMargin` instead of `SafeModeBlockMarginForTargetChain`, right?